src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java

```diff
@@ -144,7 +142,11 @@ public PluginManager configure(CNode config) throws ConfiguratorException {
             install:
             while (!plugins.isEmpty()) {
                 PluginToInstall p = plugins.remove();
-                if (installed.contains(p.shortname)) continue;
+                logger.info("Preparing to install "+ p.shortname);
+                if (installed.contains(p.shortname)) {
+                    logger.info("Plugin "+p.shortname +" is already installed. Skipping");
```

Comment:
please don't use "info" for debug info. `FINE`or even `FINER` would make sense here

Reply:
:+1: